x/project/handler.go

```diff
@@ -120,7 +133,22 @@ func handleMsgUpdateProjectStatus(ctx sdk.Context, k Keeper, bk bank.Keeper,
 	projectDoc.SetStatus(newStatus)
 	k.SetProjectDoc(ctx, projectDoc)
 
-	return sdk.Result{}
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeUpdateProjectStatus,
+			sdk.NewAttribute(types.AttributeKeyTxHash, msg.TxHash),
+			sdk.NewAttribute(types.AttributeKeySenderDid, msg.SenderDid),
+			sdk.NewAttribute(types.AttributeKeyProjectDid, msg.ProjectDid),
+			sdk.NewAttribute(types.AttributeKeyData, msg.Data.EthFundingTxnID),
```

Comment:
Attribute key should match the data that we're outputting. So for `EthFundingTxnID`, the key should be `AttributeKeyEthFundingTxnID`.

Reply:
Additionally, we are missing the new status as an attribute value. `AttributeKeyStatus`

Reply:
both has been updated .
sdk.NewAttribute(types.AttributeKeyEthFundingTxnID, msg.Data.EthFundingTxnID),
			sdk.NewAttribute(types.AttributeKeyStatus, fmt.Sprint(msg.Data.Status)),